api/proto/teleport/machineid/v1/bot.proto

```diff
@@ -63,8 +63,11 @@ message BotSpec {
 // Fields that are set by the server as results of operations. These should not
 // be modified by users.
 message BotStatus {
+  reserved 2;
+  reserved "role_role";
```

Comment:
Morbidly curious... what is `role_role` reserved for?

Reply:
My previous PR which added the protos had a mistake that I only noticed when I got around to this PR - the backwards compatibility linter requires that if I remove a field, I reserve it, to prevent a field with the same name being created in future.

Reply:
Ah, got it, thanks!